src/main/kotlin/com/vanniktech/maven/publish/MavenPublishConfigurer.kt

```diff
@@ -148,6 +148,12 @@ internal class MavenPublishConfigurer(
         if (it.name == "${plugin.name}PluginMarkerMaven") {
           // keep the current group and artifact ids, they are based on the gradle plugin id
           configurePom(it, groupId = it.groupId, artifactId = it.artifactId)
+          // workaround for https://github.com/gradle/gradle/issues/12259
+          it.pom.withXml { pom ->
+            pom.asNode().appendNode("name", publishPom.name)
```

Comment:
Is there something like `replace` or `set`. That way if Gradle were to fix their bug we won't be having two `<name>` tags?

Reply:
👍 I've added checks around these. Tested it manually by adding 
```kotlin
it.pom.withXml { pom ->
  pom.asNode().appendNode("name", "1")
  pom.asNode().appendNode("description", "2")
}
```
before which resulted in the tests failing because the pom contained 1 and 2.